jest.setup.js

```diff
@@ -0,0 +1,3 @@
+import Vue from 'vue';
+
+Vue.config.productionTip = false;
```

Comment:
Why do you want to disable those? Anything blocked you?

Reply:
Nope, this is only for the test environment. I found a bit annoying to always see the message related to the development mode. We don't need this kind of information during the tests.

> You are running Vue in development mode.
Make sure to turn on production mode when deploying for production.
See more tips at https://vuejs.org/guide/deployment.html

Reply:
But where do you see those messages? Because if it is in development mode, then it's actually expected and really helps finding some issues you might not notice.

Reply:
I saw these messages when I run the test with Jest. This change remove the warning only in Jest environment. The message is still present on Storybook for example. The flag `productionTip` [is only used on Vue startup](https://vuejs.org/v2/api/#productionTip) for displaying the message quoted above.

**Before:**

<img width="634" alt="screen shot 2017-10-24 at 10 36 33" src="https://user-images.githubusercontent.com/6513513/31932901-f15a76ee-b8a7-11e7-88c3-9514ef3eca49.png">

**After**

<img width="633" alt="screen shot 2017-10-24 at 10 36 42" src="https://user-images.githubusercontent.com/6513513/31932907-f7d58e3c-b8a7-11e7-9f59-2739a9ce1781.png">

Reply:
I think that makes sense @samouss !